src/security/PeerOptions.cc

```diff
@@ -606,6 +604,15 @@ ssl_next_proto_cb(SSL *s, unsigned char **out, unsigned char *outlen, const unsi
 #endif
 
 void
+Security::PeerOptions::updateContextOptions(Security::ContextPointer &ctx, bool setOptions) const
+{
+#if USE_OPENSSL
+    // NP: GnuTLS uses 'priorities' which are set per-session instead.
```

Comment:
I would move this GnutTLS comment lower, outside the `USE_OPENSSL` block. Adding an `#else` clause just for this comment may be a good idea.